pyjade/compiler.py

```diff
@@ -377,6 +379,9 @@ def visitAttributes(self, attrs):
 
         if temp_attrs: self.visitDynamicAttributes(temp_attrs)
 
+    def visitAndAttributes(self, andattrs):
+        self.buffer('{%% for k in %s %%} {{ k }}="{{ k[v] }}"{%% endfor %%}' % andattrs)
```

Comment:
Thanks. I didn't use `iteritems` because it does not work with python 3. Changed into using `six.iteritems` but it must be registered with each extension. So far I have only registered it with the jinja extension.